artemis/modules/joomla_scanner.py

```diff
@@ -47,6 +47,18 @@ def run(self, current_task: Task) -> None:
                 found_problems.append(f"Joomla version is too old: {joomla_version}")
                 result["joomla_version_is_too_old"] = True
 
+        # Check version via README.txt
+        response_readme = self.http_get(f"{url}/svr/www/htdocs/testing/README.txt")
+        if response_readme.status_code == 200:
+            match_readme = re.search("([0-9]+\\.[0-9]+\\.[0-9]+)", response_readme.text) # Regex for checking version
```

Comment:
I don't think such precise version is present in the README - you will have major.minor onlyy